src/components/personkort/PersonkortHeader/MaksdatoSummary.tsx

```diff
@@ -13,6 +14,9 @@ interface MaksdatoSummaryProps {
 }
 
 export function MaksdatoSummary({ maxDate }: MaksdatoSummaryProps) {
+  const utbetaltTomVerdi = maxDate.utbetalt_tom
```

Comment:
Er det noe name conflicts om vi kaller den `utbetaltTom`? 😊
```suggestion
  const utbetaltTom = maxDate.utbetalt_tom
```